### PR TITLE
Bagel Orbital Body

### PR DESCRIPTION
**Pull request by Emisse**
fuck you guys i finally set up the dev environment

### CODE REVIEW

File: Resources/Textures/Parallaxes/licences.txt [hunk 1]
```diff
@@ -3,4 +3,6 @@ Licensed under MIT at https://github.com/cpetry/TextureGenerator-Online/blob/gh-
 
 noise.png generated from https://giggster.com/guide/cloud-texture-generator/
 
-KettleParallaxNeb.png and KettleParallaxBG.png adapted from Screaming Brain Studios CC-0 https://opengameart.org/content/seamless-space-backgrounds
\ No newline at end of file
+KettleParallaxNeb.png and KettleParallaxBG.png adapted from Screaming Brain Studios CC-0 https://opengameart.org/content/seamless-space-backgrounds
+
+planet.png from tgstation at commit https://github.com/tgstation/tgstation/commit/f68af3ee3a34e07ad92125a8ee3b49193c0bc8ea
```
Comment (metalgearsloth) [inline Resources/Textures/Parallaxes/licences.txt:8]: These should be using RGA (the other ones predate it).

**Review by metalgearsloth — CHANGES_REQUESTED** [review]

**Comment by Emisse** [comment]
image
image

File: Resources/Textures/Parallaxes/licences.txt (hunk 1 above)
Comment (Emisse) [reply Resources/Textures/Parallaxes/licences.txt:8]: wtf does that mean

Comment (metalgearsloth) [reply Resources/Textures/Parallaxes/licences.txt:8]: look for attributions.yml

Comment (EmoGarbage404) [reply Resources/Textures/Parallaxes/licences.txt:8]: https://github.com/space-wizards/RobustToolbox/blob/06c64b3be8e52d52511624447cffeff551cbb262/Schemas/rga.yml

**Comment by Emisse** [comment]
ok sloth i did your bidding

**Comment by metalgearsloth** [comment]
> ok sloth i did your bidding

needs to be in a separate file called attributions.yml and needs the dash too.

**Comment by Emisse** [comment]
this is confusing

**Comment by Emisse** [comment]
ok wow im a real one

File: Resources/Textures/Parallaxes/attributions.yml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+-files: ["planet.png"]
+ license: "CC-BY-NC-SA-3.0"
+ copyright: "by Adipemdragon for tgstation, taken at commit 3839e860a7f3102b1488285d3319f61bbfe1fa51 from parallax.dmi
+ source: "https://github.com/tgstation/tgstation"
```
Comment (metalgearsloth) [inline Resources/Textures/Parallaxes/attributions.yml:4]:
```suggestion
- files: ["planet.png"]
  license: "CC-BY-NC-SA-3.0"
  copyright: "by Adipemdragon for tgstation, taken at commit 3839e860a7f3102b1488285d3319f61bbfe1fa51 from parallax.dmi
  source: "https://github.com/tgstation/tgstation"
```

**Review by metalgearsloth — APPROVED** [review]

File: Resources/Textures/Parallaxes/attributions.yml [hunk 3]
```diff
@@ -0,0 +1,4 @@
+- files: ["planet.png"]
+  license: "CC-BY-NC-SA-3.0"
+  copyright: "by Adipemdragon for tgstation, taken at commit 3839e860a7f3102b1488285d3319f61bbfe1fa51 from parallax.dmi
+  source: "https://github.com/tgstation/tgstation"
```
Comment (metalgearsloth) [inline Resources/Textures/Parallaxes/attributions.yml:4]:
```suggestion
- files: ["planet.png"]
  license: "CC-BY-NC-SA-3.0"
  copyright: "by Adipemdragon for tgstation, taken at commit 3839e860a7f3102b1488285d3319f61bbfe1fa51 from parallax.dmi"
  source: "https://github.com/tgstation/tgstation"

```

**Pull request merged**